src/library_webaudio.js

```diff
@@ -146,12 +146,13 @@ let LibraryWebAudio = {
 #endif
 
 #if WEBAUDIO_DEBUG
-    console.log(`emscripten_start_wasm_audio_worklet_thread_async() adding audioworklet.js...`);
+    console.log(`emscripten_start_wasm_audio_worklet_thread_async() adding Audio Worklet bootstrap script {{{ TARGET_BASENAME }}}.aw.js...`);
```

Comment:
Can we use `dbg()` here instead?  (Thats what I have converted all the other XXX_DEBUG macros to).

Reply:
dbg() accesses `fs.writeSync()`, which wouldn't exist in AudioWorklets. I think it's safer to keep this as is.

Reply:
`dbg()` should absolutely work everywhere.  It only goes to `fs.writeSync` under node.   If `dbg()` doesn't work in audio worklets that sounds like it would be worth fixing first since that is the primary way we do tracing/debugging (e.g. via `DEBUG_XXX` settings).

Reply:
I'll leave that for a later PR.

Reply:
https://github.com/emscripten-core/emscripten/blob/ce52783735235574811aa438e888fcab8a99e8f7/src/runtime_debug.js#L178-L190

Reply:
Ok, renamed the console.log() to dbg().

This change almost feels like one of those ones where had I proposed it first, I might be then asked to split up the console.log() -> dbg() and the functional debug print change to separate PRs to keep changes orthogonal.

Though it's there now. I agree it should work for browsers, although it will have the effect of making all debug logs appear in yellow instead of gray, since dbg() is set to print warnings.

Reply:
Ha, you are correct, I would prefer to have the `console.xxx` removal land separately :)    Not a big deal though.

I noticed that `console.error` here should probably be `err` too, but again not a big deal.

lgtm either way.  Sorry for making simple change harder than it needed to be.

Reply:
Np. Sometimes it is a "can't win either way" deal.

Btw looks like something is wrong with the code size tests. It looks as if main branch has code size values that it no longer can produce. I saw there were errors in the logs earlier about not being able to execute 'spawn' to run native closure compiler, but not sure if that error has always been there.

Way back I recall that if I rebaselined code size tests on Windows, they would not be the same on the bots, but this looks like something different.

Reply:
The codesize tests are pretty sensitive to llvm and binaryen versions.  Whenever we rebaseline we use `emsdk install tot` (or build from tip-of-tree ourselves).   I will confirm now if I can reproduce them locally.

Reply:
(I also always do `--clear-cache` before rebaselining the code size tests)